### ISSUE TITLE
Not running properly when functions defined in separate files

### ISSUE THREAD

**Issue by m0hdasif** [domagojk/serverless-console#5]
As I am having several modules and lambda functions in them. So, instead of defining it in serverless.yml file. I am declaring them in separate files and use file reference here.
Here is the structure:

```yaml
functions:
  - ${file(src/modules/hello/hello.yml)}
  - ${file(src/modules/hello/hello2.yml)}
```

This is working fine with serverless deploy command. But I tried to use the plugin, then it keeps on showing loader,not letting proceed further.
![loading add service button](https://user-images.githubusercontent.com/37996131/73600063-56512180-4571-11ea-9015-ea0ae28c202e.png)

But following code is working with plugin:
```yaml
functions:
 ${file(src/modules/hello/hello.yml)}  # '- ' is removed here but this is invalid serverless file, so can't deploy
```

**Comment by domagojk** [comment]
Hi @m0hdasif 
could you please copy/paste your output yml definition when you run "serverless print" in terminal?

**Comment by m0hdasif** [comment]
Hi @domagojk,
Here is the output of serverless print having hello.yml and hello2.yml having one function each
```yaml
service: aws-serverless
plugins:
  - serverless-plugin-typescript
  - serverless-offline-scheduler
  - serverless-offline
  - serverless-plugin-split-stacks
provider:
  stage: dev
  region: ap-south-1
  name: aws
  runtime: nodejs12.x
  environment:
    stage: dev
    region: ap-south-1
package:
  individually: true
  excludeDevDependencies: true
custom:
  splitStacks:
    perFunction: false
    perType: true
  defaultStage: dev
  stage: dev
  appName: aws-serverless
functions:
  - lambda1:
      handler: src/modules/hello/handler.lambda1
      events:
        - http:
            path: hello1
            method: get
  - lambda2:
      handler: src/modules/hello/handler.lambda2
      events:
        - http:
            path: hello2
            method: get
```
And in case of hello.yml having 2 lambda function (i.e lambda1 and lambda3) then 
```yaml
service: aws-serverless
plugins:
  - serverless-plugin-typescript
  - serverless-offline-scheduler
  - serverless-offline
  - serverless-plugin-split-stacks
provider:
  stage: dev
  region: ap-south-1
  name: aws
  runtime: nodejs12.x
  environment:
    stage: dev
    region: ap-south-1
package:
  individually: true
  excludeDevDependencies: true
custom:
  splitStacks:
    perFunction: false
    perType: true
  defaultStage: dev
  stage: dev
  appName: aws-serverless
functions:
  - lambda1:
      handler: src/modules/hello/handler.lambda1
      events:
        - http:
            path: hello1
            method: get
    lambda3:
      handler: src/modules/hello/handler.lambda3
      events:
        - http:
            path: hello3
            method: get
  - lambda2:
      handler: src/modules/hello/handler.lambda2
      events:
        - http:
            path: hello2
            method: get
```
I understand that functions property needs to be an object, but here it is an array of functions objects which works fine with serverless deployment.

**Comment by domagojk** [comment]
@m0hdasif 
Thanks for reporting! This should be fixed in version 0.5.5

**Comment by m0hdasif** [comment]
Thanks @domagojk ,
It is working in latest version
